src/lib/Libtpp/tpp_client.c

```diff
@@ -89,6 +89,7 @@
 #include <sys/time.h>
 #ifndef WIN32
 #include <stdint.h>
+#include <stdlib.h>
```

Comment:
you included this header file, but you didn't really make any other change to the file, so is this change needed?

Reply:
clang complained about the return value of malloc() being converted into an int and back to a pointer.  Since malloc() was being used, this header should have been included.